src/app/[locale]/specs/sync/page.mdx

```diff
@@ -0,0 +1,171 @@
+
+In most cases the repository data synchronized over the firehose is self-certifying (contains verifiable signatures), and consumers can verify content without making additional requests directly to account PDS instances. It is possible for services to redact events from the firehose, such that downstream services would not be aware of new content.
+
+Identity and account information is *not* self-certifying, and services may need need to verify independently. This usually means independent DID and handle resolution. Account hosting status might also be checked at account PDS hosts, to disambiguate hosting status at different pieces of infrastructure.
+
+The event message types are declared in the `com.atproto.sync.subscribeRepos` Lexicon schema, and are summarized below. A few fields are the same for all event types (except for `repo` vs `did` for `#commit` events):
+
+- `seq` (integer, required): used to ensure reliable consumption, as described in Event Streams
+- `did` / `repo`(string with DID syntax, required): the account/identity associated with the event
+- `time` (string with datetime syntax, required): an informal and non-authoritative estimate of when event was received. Intermediate services may decide pass-through this field as-is, or update to the current time
+
+### `#identity` Events
+
+Indicates that there *may* have been a change to the indicated identity (meaning the DID document or handle), and optionally what the current handle is. Does not indicate what changed, or reliably indicate what the current state of the identity is.
+
+Event fields:
+
+- `seq` (integer, required): same for all event types
+- `did` (string with DID syntax, required): same for all event types
+- `time` (string with datetime syntax, required): same for all event types
+- `handle` (string with handle syntax, optional): the current handle for this identity. May be `handle.invalid` if the handle does not currently resolve correctly.
+
+Presence or absence of the `handle` field does not indicate that it is the handle which has changed.
+
+The semantics and expected behavior are that downstream services should refresh their identity data (resolve the DID and handle) for the indicated DID. 
```

Comment:
```suggestion
The semantics and expected behavior are that downstream services should refresh their identity data from the canonical source (i.e. resolve the DID and verify the handle(s) therein) after each `#identity` event. 
```
feels like this behavioral expectation is worth being pedantically explicit about